### PR TITLE
Add AccentColor and AccentColorTest

### PR DESCRIPTION
**Pull request by xaddict**
#### Summary

Added AccentColor and AccentColorText browser compat for system colors

#### Test results and supporting details

Used BrowserStack to test browsers. They are not supported in current blink without enable-experimental-web-features so I left that out

### CODE REVIEW

File: css/types/color.json [hunk 1]
```diff
@@ -1051,6 +1051,43 @@
               "deprecated": false
             }
           },
+          "accentcolor_accentcolortext": {
+            "__compat": {
+              "description": "AccentColor, AccentColorText",
```
Comment (queengooborg) [inline css/types/color.json:1056]:
```suggestion
              "description": "<code>AccentColor</code> and <code>AccentColorText</code>",
```

**Review by queengooborg — CHANGES_REQUESTED** [review]
Hey there, thank you for your PR, and welcome to BCD!

I've just got a few requests, but this is looking good to me otherwise:
- Let's tweak the description name (see suggestion) for better formatting.
- Since the feature is supported in Chrome behind a flag, we should probably indicate it as such.  We have a `flags` property that can be utilized in this case.
- Could you provide your test code so we may verify the accuracy of the data?

**Comment by xaddict** [comment]
> Hey there, thank you for your PR, and welcome to BCD!
> 
> I've just got a few requests, but this is looking good to me otherwise:
> 
>     * Let's tweak the description name (see suggestion) for better formatting.
> 
>     * Since the feature is supported in Chrome behind a flag, we should probably indicate it as such.  We have a `flags` property that can be utilized in this case.
> 
>     * Could you provide your test code so we may verify the accuracy of the data?

1. Description name: Check
2. I'll look into the "flags" property. Is there a defaults doc page that I may have missed? Because I also want to add that all versions of Safari don't support the selected system accent color yet. They default to Safari blue (the AccentColor of Safari in iOS)
3. The test page is here: http://wpt.live/css/css-color/system-color-support.html I wrote this first only to find out that Chrome did support it (almost everything new is behind a flag in that browser). I opened that page in browserstack and flipped through versions of Chrome, Safari and Firefox on macOS and Windows.

File: css/types/color.json [hunk 2]
```diff
@@ -1051,6 +1051,43 @@
...
+          "accentcolor_accentcolortext": {
+            "__compat": {
+              "description": "<code>AccentColor</code> and <code>AccentColorText</code>",
+              "spec_url": [
+                "https://drafts.csswg.org/css-color/#valdef-system-color-accentcolor",
+                "https://drafts.csswg.org/css-color/#valdef-system-color-accentcolortext"
+              ],
+              "support": {
+                "chrome": {
+                  "version_added": false
```
Comment (Elchi3) [inline css/types/color.json:1063]: flags can be set like this:

```suggestion
                  "version_added": false,
                  "flags": [
                    {
                      "type": "preference",
                      "name": "#enable-experimental-web-platform-features",
                      "value_to_set": "true"
                    }
                  ]
```

**Review by Elchi3 — APPROVED** [review]
Thank you! 👍

**Comment by xaddict** [comment]
> Thank you! 👍

:)

Do you know the throughput time before changes show up on MDN or even caniuse?

**Comment by Elchi3** [comment]
BCD releases twice a week, dependabot then runs updates, MDN deploys multiple times a week usually as well. I would say it takes a week maximum. Likely similar for caniuse.

**Comment by yisibl** [comment]
https://issues.chromium.org/issues/40229450

**Pull request merged**